### PR TITLE
Factory bsc#1031613+bsc#102467

### PR DESCRIPTION
**Pull request by noelpower**

### CODE REVIEW

File: package/yast2-samba-server.changes [hunk 1]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Mar 30 08:42:42 UTC 2017 - nopower@suse.com
+
+- fix duplicated key warnings (bsc#1031613).
+- Don't require insall of cups for yast samba-server
```
Comment (teclator) [inline package/yast2-samba-server.changes]: typo: install

File: package/yast2-samba-server.spec [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-samba-server
-Version:        3.1.16
+Version:        3.1.17
```
Comment (teclator) [inline package/yast2-samba-server.spec]: Also for master current branch should use 3.2.X version number, I already merged a PR with `3.2.0` so just use `3.2.1`

Comment (noelpower) [inline package/yast2-samba-server.spec]: ok thanks for the heads up!, I'll try fix up this and the other comment later today

**Comment by noelpower** [comment]
fixed typo and changed version

**Pull request merged**